apps/mochi-web/pages/profile/index.tsx

```diff
@@ -23,9 +21,7 @@ const Profile: NextPageWithLayout = () => {
               className="!border-none !shadow-none"
             />
           </div>
-          <TransactionSection />
-          <PaymeSection />
-          <PaylinkSection />
```

Comment:
Can you ditch the code?

Reply:
I took out the code to merge it. Remember to do the same for similar tasks later on.